.gitignore

```diff
@@ -1,3 +1,10 @@
+.DS_Store
+package-lock.json
```

Comment:
Why we need this?

Reply:
I'm sure if we dont do this someone will commit a `package-lock.json` file by mistake at some point.

Reply:
Same can happen with any other file.
Missing file with 10K lines of code is pretty big mistake, I think we would notice it.